templates/taxi/driver_detail.html

```diff
@@ -1,18 +1,22 @@
 {% extends "base.html" %}
 
-{% block content %}
- <p><strong>
-    {{ driver.first_name }} {{ driver.last_name }}.</strong>
-    License number: {{ driver.license_number }}
-  </p>
-  {% for car in driver.cars.all %}
-    <hr>
-    <p><strong>Id:</strong>{{ car.id }}
-      <strong>Car model:</strong> {{ car.model }}
+{% if user.is_authenticated %}
```

Comment:
I guess you don't need to do this since you're using LoginRequiredMixin, because of it, there is no scenario in which you get to this page without authentication(I hope🤞)

Reply:
I think, you can add something in get request and get access to this form,I think I should
 to add this field